sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -854,6 +854,10 @@ def _add_argparse_args(cls, parser):
         '--environment_cache_millis', default=0,
         help=('Duration in milliseconds for environment cache within a job. '
               '0 means no caching.'))
+    parser.add_argument(
+        '--output_executable_path', default=None,
+        help=('Create an executable jar at this path rather than running '
+              'the pipeline.'))
```

Comment:
I can't find any reads of this parameter, other than the one below. The parameter doesn't cause outputting to the path specified. What am I missing?

Reply:
Oh, found it. It's a Java option.